newIDE/app/src/UI/SearchBar.js

```diff
@@ -205,13 +206,20 @@ const SearchBar = React.forwardRef<Props, SearchBarInterface>(
         if (
           shouldAutofocusSearchbar &&
           tagsHandler &&
-          tagsHandler.chosenTags.length === 0
+          tagsHandler.chosenTags.length === 0 &&
+          previousChosenTagsCount.current > 0
         )
           focus();
       },
       [tagsHandler, shouldAutofocusSearchbar]
```

Comment:
This may indicate also an issue where the tagsHandler was not stable between re-renders?

Reply:
Yes you're right. setting the `tagsHandler` to:
```
const tagsHandler = React.useMemo(() => ({
    add: filtersState.addFilter,
    remove: filtersState.removeFilter,
    chosenTags: Array.from(filtersState.chosenFilters),
  }), [filtersState.chosenFilters])
```
fixes the issue. I think I'll add both as it is safer.

Reply:
After thinking it through, I don't know if it's correct adding it.

Reply:
After thinking it through again, I guess memoizing things is never bad

Reply:
But indeed, tagsHandler seemed to change, but I didn't find why